_committees/21_22.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 2021-22
+
+committee:
+  - role: President
+    name: Jack Titley
+  - role: Secretary
+    name: Philippa Horn
+  - role: Treasurer
+    name: Max Nichols
+  - role: Front of House Manager
+    name: Ellie Harding
+  - role: In-House Coordinator
+    name: Em Rule
+  - role: Fringe Coordinator
+    name: Madeleine Craig
+  - role: External Relations Manager
+    name: Ylana Gibert
+  - role: Publicity Manager
+    name: Ace Edwards
+  - role: Publicity Manager
+    name: Demi Idowu
+  - role: Social and Fundraising Coordinator
+    name: Rachel Coussins
+  - role: Theatre Manager
+    name: Pete Rouse
+  - role: Company Stage Manager
+    name: Jasmine Butler
+  - role: Company Technical Director
```

Comment:
Are James and Anna the wrong way round here?

Reply:
Yes sorry ! Thank you for spotting the mistake